cmd/server-startup-msg.go

```diff
@@ -159,34 +160,36 @@ func printCLIAccessMsg(endPoint string, alias string) {
 	cred := globalServerConfig.GetCredential()
 
 	// Configure 'mc', following block prints platform specific information for minio client.
-	if isTerminal() {
-		logStartupMessage(colorBlue("\nCommand-line Access: ") + mcQuickStartGuide)
+	if color.IsTerminal() {
+		logStartupMessage(color.Blue("\nCommand-line Access: ") + mcQuickStartGuide)
 		if runtime.GOOS == globalWindowsOSName {
-			mcMessage := fmt.Sprintf("$ mc.exe config host add %s %s %s %s", alias, endPoint, cred.AccessKey, cred.SecretKey)
+			mcMessage := fmt.Sprintf("$ mc.exe config host add %s %s %s %s", alias,
+				endPoint, cred.AccessKey, cred.SecretKey)
 			logStartupMessage(fmt.Sprintf(getFormatStr(len(mcMessage), 3), mcMessage))
 		} else {
-			mcMessage := fmt.Sprintf("$ mc config host add %s %s %s %s", alias, endPoint, cred.AccessKey, cred.SecretKey)
+			mcMessage := fmt.Sprintf("$ mc config host add %s %s %s %s", alias,
```

Comment:
what is the change here?

Reply:
its just indentation to avoid the 80 column width since this file is changing anyways so add it..